xml_converter/src/xml_converter.cpp

```diff
@@ -414,45 +408,46 @@ bool filename_comp(string a, string b) {
     return lowercase(a) < lowercase(b);
 }
 
-vector<string> get_xml_files(string directory) {
+vector<string> get_xml_files(string directory, string data_directory) {
```

Comment:
This function should not take a `data_directory` as an argument as get_xml_files just needs to know what the parent directory for the xml files is. Having this argument means that in order to read the xml files you also need to copy the non-xml files to another location.